tasks/install_shell.sh

```diff
@@ -483,6 +483,26 @@ case $platform in
     filename="${collection}-release-${deb_codename}.deb"
     download_url="${apt_source}/${filename}"
     ;;
+  "Linuxmint")
+    info "Mint platform! Lets get you a DEB..."
+    case $major_version in      
+      "3")  deb_codename="stretch";;
+    esac
+    filetype="deb"
+    filename="${collection}-release-${deb_codename}.deb"
+    download_url="http://apt.puppetlabs.com/${filename}"
+    ;;
+  "LinuxMint")
```

Comment:
can you compact both cases in only one with "Linuxmint"|"LinuxMint" as pattern?

Reply:
done